.travis.yml

```diff
@@ -1,14 +1,15 @@
+sudo: false
 language: node_js
 before_install:
   - curl --location http://git.io/1OcIZA | bash -s
```

Comment:
Basically we can remove the `before_install` script if we remove [this line](https://github.com/shinnn/bl/blob/travis-yml/.travis.yml#L6). I think we don't need to test the module with Node v0.8 that is too outdated. Actually, [your other project dropped it](https://github.com/rvagg/through2/commit/0fa42c010aa58b562fd3eabfb53c0bc5ddaa60f7).